File: C#PartOne/Homeworks/03.Operators-and-Expressions-Homework/PointInACircle/PointInACircle.cs
namespace PointInACircle
{
    using System;
    using System.Globalization;
    using System.Threading;

    class PointInACircle
    {
        static void Main(string[] args)
        {
            Thread.CurrentThread.CurrentCulture = CultureInfo.InvariantCulture;

            double x = double.Parse(Console.ReadLine());
            double y = double.Parse(Console.ReadLine());
            
            double distance = Math.Sqrt((0 - x) * (0 - x) + (0 - y) * (0 - y)); //dist = math.sqrt((center_x - x) ** 2 + (center_y - y) ** 2)
            bool isInTheCirlce = distance <= 2 ? true : false; //x * x + y + y <= 4 ? true : false;
            if (isInTheCirlce)
            {
                Console.WriteLine("yes {0:F2}", distance);
            }
            else
            {
                Console.WriteLine("no {0:F2}", distance);
            }

        }
    }
}

File: JavaScript-Fundamentals/Homeworks/Operators-and-Expressions/06. Point in Circle/PointInACircle.js
function solve(args) {
    var x = +args[0],
        y = +args[1],
        radius = 2;

    var distance = Math.sqrt((0 - x) * (0 - x) + (0 - y) * (0 - y));

    var isInTheCircle = distance <= radius ? true : false;

    if (isInTheCircle) {
        return "yes " + distance.toFixed(2);
    }
    else {
        return "no " + distance.toFixed(2);
    }
}

